CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 Details changes in each release of EarlGrey. EarlGrey follows [semantic versioning](http://semver.org/).
 
+## [1.14.0](https://github.com/google/EarlGrey/tree/1.14.0) (05/30/2018)
+```
+Baseline: [e5843b3]
++ [e5843b3]: Update version number in docs and plist for 1.14.0.
+```
+
+### Enhancements
+* Update block declarations to support strict prototypes.
+* Add Swift 4 support in the gem.
+* Remove Swift 2 in the gem since Xcode 7.x is not supported anymore.
+
+### Bug Fixes
```

Comment:
any compatibility info we want to add for Xcode 9.4?

Reply:
compatibility added